website/src/components/KbSidebar/index.tsx

```diff
@@ -155,5 +158,6 @@ export default function KbSidebar() {
         </SidebarItemGroup>
       </SidebarItems>
     </Sidebar>
+>>>>>>> main
```

Comment:
I'm guessing this was left in by mistake.

Reply:
Whoops, removed.